### ISSUE TITLE
Delete button not accessible via keyboard

### ISSUE THREAD

**Issue by mschoe** [bpmn-io/bpmn-js-properties-panel#505]
__Describe the Bug__

The 'delete item ' is not accessible via keyboard if the mouse pointer is not hovering over the respective section

__Steps to Reproduce__

1. use the keyboard to create a new execution listener
2. delete the execution listener (via keyboard navigation 

__Expected Behavior__

delete item is accessible 

__Environment__

 Modeler 4.11.1-new-properties-panel
OS: Linux Mint 20.2 Cinnamon

**Comment by Skaiir** [comment]
> To fix this, we could show the delete button when any of the actions of the list item is focused (incl. the expand/collapse icon).

I second this, in my mind at least it makes sense to match element :focus (and focus-within) to mouse :hover css rules as they are the same actions just through different means. 

If we go from a purely mouse user perspective, the only difference that would occur from this change is the transition period between two drop-downs would show two delete buttons. This could be hacked away by adding some custom code which detects this case but honestly, I don't think it's worth much. What do you think from a UX perspective @andreasgeier ?

**Comment by Skaiir** [comment]
There's also a very minor visual detail regarding this, do we want highlighting on the trash icon like on mouse hover?
It just didn't look right to me. 

![image](https://user-images.githubusercontent.com/17801113/148452126-f0932e43-f421-47f9-80c5-5137b1498df7.png)

The alternative is leaving it as so:

![image](https://user-images.githubusercontent.com/17801113/148452318-e8f83d50-9821-465e-8f73-6ebddb72e884.png)

**Comment by andreasgeier** [comment]
In the artifact that @barmac created to test the new panel and templates, the states are as expected (except that keyboard focus state should have a round shape but this is really a minor problem).

keyboard focus:

![Screen Shot 2022-01-07 at 09 10 46](https://user-images.githubusercontent.com/938244/148512694-bd4e5568-3a4a-4fc7-bbf8-f93ee229d57b.png)

mouse hover:

![Screen Shot 2022-01-07 at 09 10 32](https://user-images.githubusercontent.com/938244/148512673-24722581-de69-487e-bf0b-633b6512de77.png)

This is in line with the other icon buttons where we show a focus outline for keyboard access and background color for mouseover.

**Comment by andreasgeier** [comment]
> If we go from a purely mouse user perspective, the only difference that would occur from this change is the transition period between two drop-downs would show two delete buttons.

Not sure what you mean. Can you please explain or show an example?

**Comment by Skaiir** [comment]
> Not sure what you mean. Can you please explain or show an example?

Yes so what I mean here is that you can focus one list item and hover on another, which would trigger showing the delete button on both simultaneously.

**Comment by andreasgeier** [comment]
Got it! That's not a problem in my opinion. If someone uses a keyboard and mouse at the same time (which may not be usual) this is what happened with all interactive areas.
